.gitignore

```diff
@@ -86,6 +86,9 @@ override.tf.json
 .terraformrc
 terraform.rc
 
+# Ignore providers files they are useful for local dev but not for version control as the provider version is dictated by the consuming bundles
+_providers.tf
```

Comment:
I can't get this out of my head from the docs:

https://www.terraform.io/language/modules/develop/providers#implicit-provider-inheritance

Noteably

```
Only provider configurations are inherited by child modules, 
not provider source or version requirements. 
Each module must [declare its own provider requirements](https://www.terraform.io/language/providers/requirements). 
This is especially important for non-HashiCorp providers.
```

I think if we don't want to have `version` we still should have the rest. 

```terraform
terraform {
  required_version = ">= 1.0"
  required_providers {
    massdriver = {
      source  = "massdriver-cloud/massdriver"
    }
    aws = {
      source  = "hashicorp/aws"
    }
  }
}
```

Thoughts? We yanked all the `_provider` files from these modules a while ago. If that was to solve a thing on our end I'd like to solve that, otherwise it seems like we want these in there ya?

Reply:
I added a `_providers.tf` with these contents and removed from .gitignore.

I do not know the history of why we tore them out. @chrisghill any idea?

Reply:
it seems reasonable to me to leave the version up to the bundle but code the source into the module